css/basscss.css

```diff
@@ -162,58 +137,170 @@ pre {
-blockquote p {
-  font-size: 1.25rem;
-  font-style: italic;
+.field:focus,
+.field.is-focused {
+  outline: none;
+  border-color: #0074d9;
+  box-shadow: 0 0 0 2px rgba(0, 116, 217, 0.5);
 }
 
-h1, .h1 { font-size: 2rem }
-h2, .h2 { font-size: 1.5rem }
-h3, .h3 { font-size: 1.25rem }
-h4, .h4 { font-size: 1rem }
-h5, .h5 { font-size: .875rem }
-h6, .h6 { font-size: .75rem }
+.field:disabled,
+.field.is-disabled {
+  background-color: rgba(0,0,0,.125);
+  opacity: .5;
+}
 
-.list-reset {
-  list-style: none;
-  padding-left: 0;
+.field:read-only:not(select),
+.field.is-read-only {
+  background-color: rgba(0,0,0,.125);
 }
 
-.inline       { display: inline }
-.block        { display: block }
-.inline-block { display: inline-block }
-.table        { display: table }
-.table-cell   { display: table-cell }
 
-.overflow-hidden { overflow: hidden }
-.overflow-scroll { overflow: scroll }
-.overflow-auto   { overflow: auto }
+.field.is-success {
+  border-color: #2ecc40;
+}
 
-.clearfix:before,
-.clearfix:after {
-  content: " ";
-  display: table
+.field.is-success:focus,
+.field.is-success.is-focused {
+  box-shadow: 0 0 0 2px rgba(46, 204, 64, 0.5);
 }
-.clearfix:after { clear: both }
 
-.left  { float: left }
-.right { float: right }
+.field.is-warning {
+  border-color: #ffdc00;
+}
 
-.fit { max-width: 100% }
+.field.is-warning:focus,
+.field.is-warning.is-focused {
+  box-shadow: 0 0 0 2px rgba(255, 220, 0, 0.5);
+}
+
+.field:invalid,
+.field.is-error {
+  border-color: #ff4136;
+}
+
+.field.is-error:focus,
+.field.is-error.is-focused {
+  box-shadow: 0 0 0 2px rgba(255, 65, 54, 0.5);
+}
+
+.table-light th,
+.table-light td {
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: rgba(0,0,0,.125);
+}
+
+.table-light tr:last-child td {
+  border-bottom: 0;
+}
+
+.btn {
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: bold;
+  text-decoration: none;
+  cursor: pointer;
+  display: inline-block;
+  line-height: 1.125rem;
+  padding: .5rem 1rem;
+  margin: 0;
+  height: auto;
+  border: 1px solid transparent;
+  vertical-align: middle;
+  -webkit-appearance: none;
+  color: inherit;
+  background-color: transparent;
+}
+
+.btn:hover {
+  text-decoration: none;
+  border: 1px solid rgba(0,0,0,.0625);
+  box-shadow: inset 0 0 0 20rem rgba(0,0,0,.0625);
+}
+
+.btn:focus {
+  outline: none;
+  border-color: rgba(0,0,0,.125);
+  box-shadow: 0 0 0 3px rgba(0,0,0,.25);
+}
+
+.btn:active,
+.btn.is-active {
+  box-shadow: inset 0 0 0 20rem rgba(0,0,0,.125),
+    inset 0 3px 4px 0 rgba(0,0,0,.25),
+    0 0 1px rgba(0,0,0,.125);
+}
 
-.half-width { width: 50% }
-.full-width { width: 100% }
+.btn:disabled,
+.btn.is-disabled {
+  opacity: .5;
+}
+
+
+.btn-primary {
+  color: white;
+  background-color: #0074d9;
+  border-radius: 3px;
+}
+
+.btn-outline,
+.btn-outline:hover {
+  border-color: currentcolor;
+  border-radius: 3px;
+}
+
+::-moz-focus-inner {
+  border: 0;
+  padding: 0;
+}
+
+.h1 { font-size: 2rem }
```

Comment:
I was trying to split the modules to allow for a purely class based build with no naked element styling.